test/thermodynamics/heat_transfer/general_equation_in_one_dimension_test.py

```diff
@@ -0,0 +1,42 @@
+from collections import namedtuple
+from pytest import fixture
+from sympy import sin, exp
+from symplyphysics import units, Quantity
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.thermodynamics.heat_transfer import (
+    general_equation_in_one_dimension as heat_equation
+)
+
+Args = namedtuple("Args", "rho cv k q T t x")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    rho = 1
+    cv = 1
+    k = 1
+    q = 0
+    T = sin(heat_equation.position) * exp(-1 * heat_equation.time)
+    t = 1
+    x = 1
```

Comment:
Try to look for some real numbers. This is way too synthetic.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/c83d28e5-3cf0-425d-9799-6f09fb0ad484)
The problem is that for more "real" numbers the floating point arithmetic doesn't get on well with `assert_equal`.

Reply:
This gets better if I represent the values as Rationals